src/transeq.f90

```diff
@@ -865,10 +861,8 @@ subroutine scalar_transport_eq(dphi1, rho1, ux1, uy1, uz1, phi1, schmidt, is_eve
       tb3(:,:,:) = uz3(:,:,:) * tb3(:,:,:)
     endif
     if (skewsc) then
-      ! (1-0.5) * uz.dTdz + 0.5 * (d(uz.T)dz - T.duzdz)
+      ! (1-0.5) * uz.dTdz + 0.5 * d(uz.T)dz
```

Comment:
should be (one-half) and half

Reply:
This is fixed in the updated version of this PR